lib/grape/middleware/logger.rb

```diff
@@ -80,6 +81,10 @@ def after_failure(error)
     after(error[:status])
   end
 
+  def headers
+    env['grape.request.headers']
```

Comment:
If we make this opt in, it shouldn't really matter. Just wondering how noisy this makes the log